src/components/helperFunctions.js

```diff
@@ -0,0 +1,29 @@
+export const findImageSize = (image, imageSizes) => {
```

Comment:
Her finner du et sett med bilder som tilhører denne bilde-URLen. Så sett denne metoden i flertall for å forhindre forvirring.